Makefile

```diff
@@ -170,9 +166,10 @@ e2e-coverage: ginkgo
 integration-test-istio: ginkgo k3d | test-matchers provision-test-env ## Provision k3d cluster, deploy development variant and run integration tests with istio.
 	ISTIO_VERSION=$(ISTIO_VERSION) hack/deploy-istio.sh
 	IMG=k3d-kyma-registry:5000/telemetry-manager:latest make deploy-dev
-	$(GINKGO) run --tags e2e -v --junit-report=junit.xml --label-filter="istio" ./test/integration
+	$(GINKGO) run --tags istio -v --junit-report=junit.xml ./test/integration/istio
```

Comment:
The access log tests are now having the "logging" label? Will they be executed in this make target?